src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
```

Comment:
Avoid using unnecessary operations. Here, you are creating array of values but use it only once. You can directly check the robot object in the for loop